index.html

```diff
@@ -0,0 +1,448 @@
+                don’t hestiate to contact me.</p>
+            <div class="supporting-text">
+                <a class="connect" href="#">LET’S CONNECT</a>
+            </div>
+
+            <ul class="frame1">
+                <li><a href="#"><img class="github2" src="images/github.png"></a></li>
+                <li><a href="#"><img class="linkedin2" src="images/Linkedin icon.png"></a></li>
+                <li><a href="#"><img class="vector2" src="images/Vector.png"></a></li>
+                <li><a href="#"><img class="twitter2" src="images/twitter.png"></a></li>
+                <li><a href="#"><img class="m2" src="images/m.png"></a></li>
+            </ul>
+
+            <button type="button" class="resume">
+                <span>Get my resume</span>
+            </button>
+
+        </div>
+
+        <div class="skill-list">
+            <div class="frame12">
+                <ul class="frame7">
+                    <li class="lang">Languages</li>
+                    <li class="arrow-down"><i class="fa-solid fa-chevron-down"></i></li>
+                </ul>
+            </div>
+
+            <div class="language-list">
+                <ul class="frame9">
+                    <li class="js-ellipse">
+                        <a href="#"><img src="images/js.png"></a>
+                    </li>
+                    <li class="script">JavaSript</li>
+                </ul>
+                <ul class="frame10">
+                    <li class="html-ellipse">
+                        <a href="#"><img src="images/html.png"></a>
+                    </li>
+                    <li class="script2">HTML</li>
+                </ul>
+                <ul class="frame11">
+                    <li class="css-ellipse">
+                        <a href="#"><img src="images/css.png"></a>
+                    </li>
+                    <li class="script3">CSS</li>
+                </ul>
+            </div>
+        </div>
+        <ul class="frame-8">
+            <li class="script4">Frameworks</li>
+            <li class="arrow-right"><i class="fa-solid fa-chevron-right"></i></li>
+        </ul>
+     <hr class="hr">
+        <ul class="frame-9">
+            <li class="script5">Skills</li>
+            <li class="arrow-right2"><i class="fa-solid fa-chevron-right"></i></li>
+        </ul>
+        <hr class="hr">
+
+    </section>
+    <section class="contact-form">
+             
+                <div class="contain" >
+        <h2 id="contact-me">Contact me</h2>
+        <p class="contact-p">If you have an application you are interested in developing, a feature that you need built
+            or a project that needs coding. I’d love to help with it </p>
+
+        <form action="https://formspree.io/f/mdojqvjo" method="POST">
+            <input type="text" size="30" id="name" placeholder="Yeremias" name="name" required>
+            <input type="email" size="30" id="email" placeholder="natajayanj14@gmail.com" name="email" required>
```

Comment:
- [ Required ] Please consider removing the size constraint on the `email` field, this might prevent someone who has an email that is more than 30 characters long from submitting the form. The size validation should be placed only on the `name` field as specified in the project requirements.